FrameGrabberSimulator/FileCopier.cs

```diff
@@ -66,50 +80,30 @@ private void ModifyOutliers(List<double> sortedDistribution)
 
         private bool IsOutlier(double value)
         {
-            return value < -1;
+            return value < -1 || value > 1;
         }
 
         private List<int> CreateNormalDistributedDelayList(List<double> distribution, int mean)
         {
             List<int> delayList = new List<int>();
-
             foreach (var value in distribution)
             {
                 var newValue = mean - Math.Abs(value * mean);
                 delayList.Add((int)Math.Round(1000 / newValue));
             }
-
             return delayList;
         }
 
-
-        public void CopyFiles(string sourceDir, string targetDir)
+        private void SimulateProjectionCreation(int i)
         {
-            Directory.CreateDirectory(targetDir);
-            string[] files = Directory.GetFiles(sourceDir, _fileType);
-
-            for (int i = _startProjection; i < _amountOfProjections; i++)
+            if (_modeSettings == _mode)
```

Comment:
This is some what insufficient since you are generating the list even if you just want the normal frequency. 
Consider your design - I believe that you should extract the "simulate projection creation behaviour" and let a factory decide depending on the mode which one should be returned.